tests/tests_common_test.go

```diff
@@ -204,3 +208,25 @@ func getResourceKey(obj controllerutil.Object) client.ObjectKey {
 		Name:      obj.GetName(),
 	}
 }
+
+func expectUserCan(user string, groups []string, verb, resNamespace, resGroup, resVersion, resKind, resName, resSubresource string) {
```

Comment:
It can be confusing to have a function with 9 parameters. Can you use one `*authv1.ResourceAttributes` instead of the last 7 string parameters?

Reply:
Sure, will handle it in #84